src/test/java/core/basesyntax/MatrixOperationTest.java

```diff
@@ -1,8 +1,81 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
-public class HelloWorldTest {
+import org.junit.Assert;
+import org.junit.Test;
 
-}
\ No newline at end of file
+import java.util.Arrays;
+
+public class MatrixOperationTest {
+    private static final int[][] FIRST_TEST_MATRIX = {
+            {2, 31, 6, 7},
+            {15, 42, 1, 0},
+            {9, 7, 12, 19},
+            {55, 34, 1, 10}
+    };
+    private static final int[][] SECOND_TEST_MATRIX = {
```

Comment:
what's the conceptual difference between first and second matrix?

Reply:
there isn't the conceptual difference
only different size of matrices